js/internal-template-events.js

```diff
@@ -0,0 +1,10 @@
+(function ($) {
+  /*Submit url to rate form*/
+  $(document).on('submit', '#RateThisPageForm', function () {
```

Comment:
While the style guide is somewhat inconsistent on this, I think event handlers should be extracted as named functions rather than using anonymous functions. For example:

// Bad
```JavaScript
$('p').on('click', function(e) { ... });
```
// Good
```JavaScript
var doSomething = function() { ... }

$('p').on('click', doSomething);
```

I'm only going to comment this in once, but it applies to everything. If you disagree, I'm open to discussion.